_commands/basics/who.md

```diff
@@ -3,22 +3,65 @@
 
 who
 --
-TODO: Add documentation for this command by submitting a pull request.
 <!-- one line explanation would go here -->
+## Description
+Displays list of all users currently logged on 
 
 <!-- minimal example -->
+### Example Output 
 ~~~ bash
-*short* example of command usage and output
+$ Username console  Oct 12 08:20 
+$ Username ttys000  Oct 20 09:35 
 ~~~
 
-<!--more-->
+ The who utility displays a list of all users currently logged on, showing for each user the login name, tty
+ name, the date and time of login, and hostname if not local.
 
-### Useful Options / Examples
+#### Available options with Examples:
+ `-a`     Same as `-bdlprTtu`. This command processes the system database used for user information.
+ 
+ `-b`    Time of last system boot. 
+ 
+             $ who -b 
```

Comment:
This can be de-indented and surrounded with "\~\~\~ bash" before it and "\~\~\~" after it, much like the example towards the beginning of the file.